src/net/fd_wasi.go

```diff
@@ -0,0 +1,78 @@
+//go:build wasi
+// +build wasi
+
+package net
+
+import (
+	"os"
+	"syscall"
+	"time"
+)
+
+// Network file descriptor.
```

Comment:
```suggestion
// wasiCon implements Con with a file returned by syscall.SockAccept.
```

Reply:
This is directly copy-pasted and adapted from https://github.com/golang/go/blob/a0441c7ae3dea57a0553c9ea77e184c34b7da40f/src/net/fd_posix.go#L16-L17, that's the approach I saw throughout this project so I followed that.

Changing this comment would make it inconsistent with upstream, which means it would not be as trivial to sync this with upstream by copy-pasting.

If we're changing this, then we should change the whole implementation as well, since the way this is designed in the standard library is not necessarily the best approach here given the set of use cases this project tries to cover.

Reply:
ok understand what you mean. I think in another world I would probably comment things like this in the source, but ack that's not the practice. I'll dismiss my feedback as most of them are me being curious why things are defining unused symbols or otherwise. People who know the conventions already somehow won't need to see comments like this.